modules/javafx.controls/src/main/java/javafx/scene/control/cell/TextFieldTreeCell.java

```diff
@@ -189,22 +189,20 @@ public final void setConverter(StringConverter<T> value) {
 
     /** {@inheritDoc} */
     @Override public void startEdit() {
-        if (! isEditable() || ! getTreeView().isEditable()) {
+        super.startEdit();
+        if (!isEditing()) {
             return;
         }
```

Comment:
similar to ChoiceBox/ComboBoxTreeCell, except that a similar test fails both before/after the fix

Reply:
hm interesting enough this tree cell has no `treeItem = null` guard (unlike the others). TreeCells are very weird 😃